common/app/model/SupportedUrl.scala

```diff
@@ -13,7 +13,11 @@ object SupportedUrl {
 
   def fromFaciaContent(fc: PressedContent): String =
     fc match {
-      case curatedContent: CuratedContent => s"/${curatedContent.properties.href.getOrElse(fc.card.id)}"
+      case curatedContent: CuratedContent =>
+        s"/${curatedContent.properties.webUrl
+          .map(webUrl => webUrl.replaceFirst("^[a-zA-Z]+://www.theguardian.com/", ""))
```

Comment:
```suggestion
          .map(webUrl => webUrl.replaceFirst("https://www.theguardian.com/", ""))
```

Could we just use this for clarity? Is there use cases that being more generic about the protocol covers?

Reply:
I have pushed the simplification to this regex. I am not aware of any use cases to be honest, I just used the same regex as here: https://github.com/guardian/frontend/blob/main/common/app/utils/ShortUrls.scala#L25.